src/lib/utils/functions/sqlFunctions.ts

```diff
@@ -84,14 +84,17 @@ export const getWhereInsertIndex = (tokens: string[][]): number => {
   return targetIndex
 }
 
-// This will construct a sql query by adding the conditions in searchParams
-// to the WHERE clause, preserving all other clauses.
-// If the searchParams are not defined, this will simply return the given sql.
+/**
+ * This will construct a sql query by adding the conditions in searchParams
+ * to the WHERE clause, preserving all other clauses.
+ * If the searchParams are not defined, this will simply return the given sql.
+ */
 export const insertConditionsFromSearchParams = (
   sql: string,
   searchParams?: KeyValue,
   operator: SQLOperator = 'LIKE',
 ) => {
+  // TODO: Replace SQL manipulation with QueryFilters. See PORTALS-2157
```

Comment:
This is the only other place where we're using sql-parser. I think it's how we get the locked facet from a DetailsPage. I think we could probably add a QueryFilter rather than manipulating SQL, but that investigation/work are tracked in a new ticket.